Node/core/src/bots/ChatConnector.ts

```diff
@@ -955,8 +1001,22 @@ export class ChatConnector implements IConnector, IBotStorage {
         options.headers['User-Agent'] = USER_AGENT;
     }
 
-    private addAccessToken(options: request.Options, cb: (err: Error) => void): void {
+    private addAccessToken(options: request.Options | any, cb: (err: Error) => void): void {
```

Comment:
nit: if you do ` x | any` it will always be `any`. Avoid using `any`.

Reply:
fixed